usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -239,6 +242,19 @@ def handle(self, *args, **options):
                     cursor.execute(create_temp_sql)
                     logger.info(f"{temp_table} created.")
 
+                    # If there are vectors, add the triggers that will populate them based on other calls
+                    logger.info(f"Adding tsvector triggers for the following columns: {list(tsvectors.keys())}")
+                    for tsvector_name, derived_from_cols in tsvectors.items():
```

Comment:
Does this need to be guarded by a check that `tsvectors` is not `None`. The `.get(...)` above could allow it to be. And if we instead default it to empty list, the log statement should be moved into the for loop or otherwise dealt with so that we don't log about an empty dict of tsvectors

Reply:
Updating it to `tsvectors = table_spec.get("tsvectors") or []` which should account for if it's not included *and* if they do include it in the TABLE_SPEC but set it to `None`.